src/clr-angular/utils/_theme.dark.clarity.scss

```diff
@@ -519,8 +448,8 @@ $clr-forms-checkbox-disabled-mark-color: #000;
 // Radio
 $clr-forms-radio-label-color: #adbbc4;
 $clr-forms-radio-selected-shadow: $clr-forms-checkbox-checked-shadow;
-$clr-forms-radio-focused-shadow: 0 0 2px 2px $clr-outline-color;
-$clr-forms-radio-disabled-background-color: $clr-form-disabled-background-color;
+$clr-forms-radio-focused-shadow: 0 0 2px 2px $clr-link-active-color;
+$clr-forms-radio-disabled-background-color: #000;
```

Comment:
Should this be `$clr-color-neutral-1000` ?

Reply:
I just checked on this one, the rest of the vars in the theme don't use the color vars directly. It looks like some of the other scss var files would have to be imported? Not sure if that is something we want to do in this PR since it's already quite large.